src/activities/menu/Menu.qml

```diff
@@ -77,6 +78,8 @@ ActivityBase {
 
     // @cond INTERNAL_DOCS
     property string url: "qrc:/gcompris/src/activities/menu/resource/"
+    property int  cursorPosition: 0
+    property  string inputText: ""
```

Comment:
yes, because it is embedded. Then, you can set the TextInput.text value to inputText (maybe rename it searchedText?), so you won't have to update it manually. The aim is to have only one variable to update